src/providers/cloudflare-stream.js

```diff
@@ -0,0 +1,166 @@
+const { ok } = require('node:assert');
+const crypto = require('node:crypto');
+const AbstractFileTransfer = require('ms-files-transport');
+const Cloudflare = require('cloudflare');
+const { HttpStatusError } = require('common-errors');
+
+const {
+  FILES_CONTENT_LENGTH_FIELD,
+  FILES_NAME_FIELD,
+} = require('../constant');
+
+const NotImplementedHttpError = new HttpStatusError(501, 'Method \'copy\' is not implemented');
+
+const toBase64 = (value) => Buffer.from(value).toString('base64');
+const nowPlusSeconds = (seconds) => (new Date(Date.now() + 1000 * seconds)).toISOString();
+const nowPlus30Days = () => nowPlusSeconds(2592000);
+const arrayBufferToBase64Url = (buffer) => Buffer.from(buffer).toString('base64url');
+const objectToBase64url = (payload) => arrayBufferToBase64Url(JSON.stringify(payload));
```

Comment:
https://www.npmjs.com/package/safe-stable-stringify use this instead of JSON.stringify